app/models/user.rb

```diff
@@ -7,4 +7,14 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
   :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :active_followings, class_name: "Following",
+                               foreign_key: "follower_id",
+                               dependent: :destroy
+  has_many :passive_followings, class_name: "Following",
```

Comment:
The table is Following. active following and passive following is an alias given so we will not be refering to the same thing when we refer to followers or followed. Refer to tutorial at https://www.railstutorial.org/book/following_users